src/js/views/admin/actions/actions_views.js

```diff
@@ -265,9 +265,70 @@ const AttachmentComponent = Droplist.extend({
   },
 });
 
+const RoleItemTemplate = hbs`<a{{#if isSelected}} class="is-selected"{{/if}}>{{matchText name query}} <span class="program-actions__role">{{matchText short query}}</span></a>`;
```

Comment:
I know we don't have a great place to put these common elements right now, but this really isn't action related.  Could probably just go into the clinician views for now